test/tools/yulInterpreter/Interpreter.cpp

```diff
@@ -106,11 +106,27 @@ void Interpreter::operator()(ForLoop const& _forLoop)
 	while (evaluate(*_forLoop.condition) != 0)
 	{
 		(*this)(_forLoop.body);
```

Comment:
Might want to add an assertion that loop state is default before we enter the body - or could also set it there and after the loop as opposed to the two other places.

Reply:
i've moved it on top, but left the very bottom one, because: i need the last in order to make the following work:
```
for {} cond {}
{
    for {} cond {}
    {
        break
    }
    // without reset, it would break out here to, due to operator()(Block const&)'s impl
    for {} cond {}
    {
        // ...
    }
}
```